datarequest.py

```diff
@@ -273,6 +304,10 @@ def api_datarequest_submit(ctx, data, previous_request_id):
     coll_path = "/{}/{}/{}".format(user.zone(ctx), DRCOLLECTION, request_id)
     file_path = "{}/{}".format(coll_path, DR_FILENAME)
 
+    # Validate data against schema
+    if not datarequest_data_valid(ctx, data, DR):
+        return api.Error("validation_fail", "Data did not pass validation.")
```

Comment:
Specify what data could not pass validation?

Validation of datarequest failed?